packages/compiler/testing/test-host.ts

```diff
@@ -140,32 +158,32 @@ export async function createTestHost(): Promise<TestHost> {
     logSink: NodeHost.logSink,
     mkdirp: async (path: string) => path,
   };
+}
 
-  // load standard library into the vfs
-  for (const [relDir, virtualDir] of [
-    ["../../lib", "/.cadl/dist/lib"],
-    ["../../../lib", "/.cadl/lib"],
-  ]) {
-    const dir = resolvePath(fileURLToPath(import.meta.url), relDir);
-    const entries = await readdir(dir, { withFileTypes: true });
-    for (const entry of entries) {
-      const realPath = resolvePath(dir, entry.name);
-      const virtualPath = resolvePath(virtualDir, entry.name);
-      if (entry.isFile()) {
-        switch (getAnyExtensionFromPath(entry.name)) {
-          case ".cadl":
-            const contents = await readFile(realPath, "utf-8");
-            virtualFs.set(virtualPath, contents);
-            break;
-          case ".js":
-          case ".mjs":
-            jsImports.set(virtualPath, import(pathToFileURL(realPath).href));
-            virtualFs.set(virtualPath, ""); // don't need contents.
-            break;
-        }
-      }
-    }
+export const StandardTestLibrary: CadlTestLibrary = {
+  name: "@cadl-lang/compiler",
+  packageRoot: resolvePath(fileURLToPath(import.meta.url), "../../../"),
+  files: [
+    { virtualPath: "/.cadl/dist/lib", realDir: "./dist/lib", pattern: "*" },
+    { virtualPath: "/.cadl/lib", realDir: "./lib", pattern: "*" },
+  ],
+};
+
+class TestHostError extends Error {
+  constructor(message: string, public code: "ENOENT" | "ERR_MODULE_NOT_FOUND") {
+    super(message);
   }
+}
+
+function resolveVFsPath(path: string) {
```

Comment:
`resolveVfsPath` or `resolveVFSPath`?

Reply:
I renamed this to resolveVirtualPath in #207 :) I was holding that back to avoid merge conflicts, but regretting my choice now, lol.

Reply:
I'll wait until the projection and your Pr are merged so I resolve that conflict

Reply:
Thanks, and I'm waiting on your emit PR too.

I think fair FIFO order is

1. projections 
2. emit
3. testable LS
4. this PR

cc @bterlson

Reply:
sounds good